### DISCUSSION TITLE
main menu on linux windows like

### DISCUSSION THREAD

**Discussion by jakoujan** [apache/netbeans#4912] [Q&A]
Hi as a old user in netbeans i allways use in all OS this IDE but i note that the interface is diferent between linux and windows personally i use a most of time linux but in some times i use windows for customer requirements.

is there some way that i can have the same main menu position on title bar instead of the main title bar on linux?

![image](https://user-images.githubusercontent.com/11701794/199873201-52a25c11-8af4-4302-85c5-5a9351a35a15.png)

this is the main menu on window i was wondering if this i could be possible to have on linux thanks in advance.

**Comment by neilcsmith-net** [comment]
@matthiasblaesing and @DevCharly might have thoughts on this?  I believe it's technically possible now with FlatLaf, but support probably needs adding in NetBeans itself?

**Reply by jakoujan** [reply]
this could be a good improvement into netbeans UI this give a fresh look to the IDE and in terms of usability gives a better use of spacen thanks for the response. i hope it can be make for a next releases.

**Reply by jakoujan** [reply]
i was seeing in the flatLaf configuration that have this options disabled, i don't know why?

![Captura desde 2022-11-04 10-08-40](https://user-images.githubusercontent.com/11701794/200023223-8f2d544a-b647-423e-be9c-bbd0a2357aff.png)

**Reply by matthiasblaesing** [reply]
FlatLAF gained the ability to draw the combined header on linux in the most recent release. It requires that the LAF is tasked to draw the window decorations. The option, that switches that behavior must be set as a system property before the main window is drawn (at least to my understanding).

This indeed needs someone to take fiddle into the right point and rework the checkbox activation logic.

**Reply by jakoujan** [reply]
this improvement could be integrated in some future release? i've never see the netbeans code, how colud i help in order to get this feature?
